### ISSUE TITLE
[BUG] Maps - "Double tap to activate" - Specify functionality intention

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5498]
**Describe the bug**
When a TalkBack user is gesturing through Maps after "Campus Buildings" filter is applied, after "Amenities" they hear  "Double tap to activate" and then if they double tap they hear a "bmp" sound only. 

**To Reproduce**
1. Using TalkBack on Android, Go to 'Maps'
2. Make sure the "Campus Buildings" filter is set
3. Gesture through past '..Amenities.'

**Expected behavior**
If a TalkBack user hears "Double tap to activate" then they double tap, something should happen. 

**Actual behavior**
When a TalkBack user is gesturing through Maps after "Campus Buildings" filter is applied, after "Amenities" they hear  "Double tap to activate" and then if they double tap they hear a "bmp" sound only. 

_**What is the functionality that is supposed to happen when double tapping, what is the user supposed to hear next and in what order?**_

**Screenshots**

https://github.com/user-attachments/assets/01450c78-4459-43f8-9d0d-b1a6e482dc10

 
**Smartphone (please complete the following information):**
•	OS: Android Version 15
•	Device: Google Pixel 8a
•	Browser: Chrome 141.0.7390.123
- Version 7.3.16

**Additional context**

**Comment by TodorBachvarov** [comment]
The 'Double tap to activate' message observed for the Google Map widget is not directly tied to an accessibility action set by our code. Instead, it stems from the native Google Maps widget's inherent complexity and design which gives it its own independent accessibility. The entire map is tappable because it uses these taps to determine whether you're interacting with a marker, or focusing on a specific area. Because the native platform sees this as an inherently tappable element, it automatically provides the 'Double tap to activate' accessibility message, even without a specific action defined by our code. There are two possible solutions to consider: 
1. Disable the whole semantics for the Google Map, which would remove the accessibility exposure of focusing and tapping on markers and other interactive elements. In my opinion, this solution would greatly reduce the functionality that the blind user can use with our view. 
2. Add a default tap action for the map (since we can't remove it), so that when a blind user hears 'Double tap to activate,' the action is performed, which can help prevent frustration. In this case we will have to decide what this default action should be. 
We will need to decide which trade-off is worse and determine the appropriate path forward. 
@acovher2, I'll look forward for advice on this matter.

**Comment by acovher2** [comment]
@TodorBachvarov Just reading your N.B on #5500 and discussing with a colleague. This is the way I have seen it on other sites. In this case if the screen reader users can't interact with results on map, they would just need to gesture past search results and focus could land on search result amount in the tray and expand the tray for them to continue gesturing through the results. RIght? 

Example: If the Talkback user selects Map filter of ADA Entrance and applies. Results are announced by landing focus to the Selected Filter's results. 

Like this:

![Image](https://github.com/user-attachments/assets/bd8863e3-9203-4eae-8ca3-7c2edd9943f8)
